specs/experimental/l2_blob_tx.md

```diff
@@ -0,0 +1,47 @@
+<!-- omit in toc -->
+# L2 Blob Transaction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Motivation](#motivation)
+- [How It Works](#how-it-works)
+- [Reference Implementation](#reference-implementation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+The proposal aims to integrate BLOB transaction support into the OP Stack. BLOB transactions are gaining popularity on Layer 1 (L1), with applications including BLOB inscriptions, Rollups (as an optional feature), and EthStorage. However, none of the L2s (including OP Stack) supports BLOB transactions, resulting in high migration costs for these projects. To harvest the reduced gas costs of L2s with minimized migration costs of applications using BLOB transactions, it is of great value in OP Stack to support L2 BLOB transactions.
+
+To further reduce the DA cost of L2 BLOB transactions of OP Stack, the proposal offers a hybrid DA solution, where the L2 BLOBs may use a different DA (e.g., DA committee or DA challenge) as that of L2 calldata (generally L1 DA).  This better serves the applications with demands on different data values with lower costs:
+- For transactions with high-value data (e.g., swap in Uniswap), the users can use non-BLOB L2 transactions.
```

Comment:
If user use non-BLOB L2 transactions, how it can be the option of `To further reduce the DA cost of L2 BLOB transactions`

Reply:
As I understand it, the proposal suggests using an alternative data availability (DA) solution for L2 BLOB transactions instead of Ethereum’s native DA to further reduce the cost of these transactions.